frontend/src/components/UserUpdate.vue

```diff
@@ -50,13 +77,24 @@ export default {
         .finally(() => {
           window.scrollTo(0, 0);
         });
-    }
+    },
+    getMailboxClients: async function () {
+      //get both SFSD and HSCIS
+      this.MailboxClients = [];
+      let clients = await ClientsRepository.get();
+      clients = clients.data;
+      clients.map((client) => {
+        if (this.MailboxClientNames.includes(client.name)) {
+          this.MailboxClients.push(client);
+        }
+      });
+    },
```

Comment:
I think we could move all of this functionality into UserMailboxAuthorizations.vue
that way we can keep the names of clients with mailboxes only in mailbox related components, and don't need to pass any props between the components